src/test/java/com/test/Flipkart1.java

```diff
@@ -46,14 +49,14 @@ public Boolean data(String product, String filter2) {
         driver.manage().window().maximize();
         driver.get("https://www.flipkart.com/");
         Action a = new Action(driver);
```

Comment:
Use a meaningful name for both the class and object like - FlipkartPage or something